# 裁判警告

Header header

# 警告等级
# 黄牌
uint8 LEVEL_YELLOW = 1
# 红牌
uint8 LEVEL_RED    = 2
# 判负
uint8 LEVEL_DEFEAT = 3
uint8 level

# 补弹机器人类型,若没有机器人在补弹,则此项无意义
# 英雄机器人
uint8 TYPE_HERO = 1
# 工程机器人
uint8 TYPE_ENGINEER = 2
# 步兵机器人3号
uint8 TYPE_STANDARD3 = 3
# 步兵机器人4号
uint8 TYPE_STANDARD4 = 4
# 步兵机器人5号
uint8 TYPE_STANDARD5 = 5
# 空中机器人
uint8 TYPE_AERIAL = 6
# 哨兵机器人
uint8 TYPE_SENTRY = 7
# 飞镖机器人
uint8 TYPE_DART = 8
# 雷达站
uint8 TYPE_RADER = 9
uint8 robotType
